plugins/cockpit/pom.xml

```diff
@@ -164,6 +164,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-core-resources</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+              <includeGroupIds>io.camunda</includeGroupIds>
+              <includeArtifactIds>camunda-7-to-8-data-migrator-core</includeArtifactIds>
+              <includes>
+                mapper/*.xml,
+                db/properties/*.properties,
+                io/camunda/migrator/impl/persistence/IdKeyDbModel.class,
+                io/camunda/migrator/impl/persistence/IdKeyMapper.class,
+                io/camunda/migrator/impl/persistence/IdKeyMapper$TYPE.class
+              </includes>
```

Comment:
It's not ideal, but it makes the plugin standalone. When striving for production readiness, we should add at least an integration test that detects when something breaks here.

A fat jar wouldn't work in all scenarios: E.g., Spring Boot classes would lead to conflicts when deploying on Camunda Run.

Reply:
We could recheck if more scopes can now be `provided` since we just copy some classes and resources over.